### ISSUE TITLE
Give search results more usability

### ISSUE THREAD

**Issue by hinerm** [imagej/napari-imagej#53]
It would be convenient if:

- Double-clicking a search result created the widget for it
- Right-clicking a search-result opened a context menu with the same "staging" options (Run, Help, Source, Batch)

**Comment by hinerm** [comment]
Also, I would put `Classes` last, personally.. that seems the least helpful of the results.

**Comment by gselzer** [comment]
> Also, I would put `Classes` last, personally.. that seems the least helpful of the results.

Yeah, I'm not sure why it shows up first...the napari search results seem to show up in alphabetical order.

**Comment by hinerm** [comment]
>Also, I would put Classes last, personally.. that seems the least helpful of the results.

Addressed in https://github.com/imagej/napari-imagej/pull/122!
